### PR TITLE
docs: Updated spell schema's damage property

### PR DESCRIPTION
**Pull request by ThomasGCarpenter**
## What does this do?
Spell damage schema models added to common.yml file allow for more accurate spell damage schema. 

## How was it tested?
Followed  swagger README instructions to set up local development environment which ran successfully after adding new openapi documentation. Then validated and bundled swagger with no errors. Then manually tested various http requests on Postman successfully.   

## Is there a Github issue this is resolving?
Yes. This resolves issue #289. 

## Was any impacted documentation updated to reflect this change?
The changes were made exclusively to the api documentation.  

## Here's a fun image for your troubles
random photo - update me

### CODE REVIEW

File: src/swagger/schemas/common.yml [hunk 1]
```diff
@@ -257,3 +257,23 @@ area-of-effect-model:
...
+damage-at-slot-level-model:
+  description: |
+    'Spell Damage'
+  type: object
+  properties: 
+    damage_at_slot_level:
+      type: object
+      additionalProperties: true
+    damage_type:
+      $ref: './combined.yml#/APIReference'
```
Comment (bagelbits) [inline src/swagger/schemas/common.yml:279]: Can you put these in `spell.yml` for now? I don't think these shapes get used outside of Spells for right now.

File: src/swagger/swagger.yml [hunk 2]
```diff
@@ -323,3 +323,5 @@ components:
 
   schemas:
     $ref: './schemas/combined.yml'
+
+
```
Comment (bagelbits) [inline src/swagger/swagger.yml:327]:
```suggestion
```

Comment (bagelbits) [inline src/swagger/swagger.yml:327]:
```suggestion
```

**Review by ThomasGCarpenter — COMMENTED** [review]
Hello, thank you for the feedback. I added the spell damage models to the spell.yml file.

File: src/swagger/schemas/spell.yml [hunk 3]
```diff
@@ -1,69 +1,86 @@
...
+damage-at-slot-level-model:
+  description: |
+    'Spell Damage'
+  type: object
+  properties: 
+    damage_at_slot_level:
+      type: object
+      additionalProperties: true
+    damage_type:
+      $ref: './combined.yml#/APIReference'
```
Comment (bagelbits) [inline src/swagger/schemas/spell.yml:86]:
```suggestion
      $ref: './combined.yml#/APIReference'

```

**Review by bagelbits — APPROVED** [review]

**Pull request merged**